android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/ConfigUriBeacon.java

```diff
@@ -0,0 +1,287 @@
+package org.physical_web.physicalweb;
+
+/*
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.net.URISyntaxException;
+
+public class ConfigUriBeacon extends UriBeacon {
```

Comment:
We don't need this whole class...just pull out the one utility method you use in throw it in one of the utility classes we have.